Chore: Add check for flaky tests in PR checklist (#2)
inoqo/.github: .github/pull_request_template.md

## Pre-Deploy TODOs

- [ ] I have tested migrations on real data

> - [ ] I checked I have no flaky tests (e.g. relying on query results order)

Reviewer: ```suggestion
- [ ] I checked for common causes of flaky tests (e.g. relying on query results order)
```
